vita3k/config/src/config.cpp

```diff
@@ -318,6 +347,16 @@ ExitCode init_config(Config &cfg, int argc, char **argv, const Root &root_paths)
         po::store(po::command_line_parser(argc, argv).options(all).positional(positional).run(), var_map);
         po::notify(var_map);
 
+#ifdef USE_VULKAN
+        if (to_lower(backend) == "vulkan")
+            command_line.backend = 1;
+        else
+#endif
+            if (to_lower(backend) == "opengl")
```

Comment:
Fucked up

Reply:
You mean the indenting? Clang-format did this so I though I'd trust it :/

I agree it looks ugly... I'll try to figure something out.